extensions/spring-xd-extension-batch/src/main/java/org/springframework/batch/step/tasklet/x/AbstractProcessBuilderTasklet.java

```diff
@@ -138,13 +202,29 @@ public RepeatStatus execute(StepContribution contribution, ChunkContext chunkCon
 
 	@Override
 	public void beforeStep(StepExecution stepExecution) {
+		exitCode = -1;
```

Comment:
I'm pessimistic so -1 seems better until we have success reported from the process